### ISSUE TITLE
[GENERAL]: `TypeError: 'NoneType' object is not subscriptable` when doing `relay_set` over jsonrpc

### ISSUE THREAD

**Issue by jaryn** [UniPiTechnology/evok#57]
### Prerequisites
Running evok 56017202d80f12c986e5522bdfcb71b6abd989a9

### Log files
```
In [56]: from jsonrpclib import Server
    ...: Server("http://192.168.2.241/rpc").relay_set('2_02', True)
    ...: 
    ...: 
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-56-c825eeb3a4c4> in <module>()
      1 from jsonrpclib import Server
----> 2 Server("http://192.168.2.241/rpc").relay_set('2_02', True)
      3 

/usr/lib/python3.6/site-packages/jsonrpclib/jsonrpc.py in __call__(self, *args, **kwargs)
    648                                 "arguments (according to JSON-RPC spec.)")
    649         if args:
--> 650             return self.__send(self.__name, args)
    651         else:
    652             return self.__send(self.__name, kwargs)

/usr/lib/python3.6/site-packages/jsonrpclib/jsonrpc.py in _request(self, methodname, params, rpcid)
    531         response = self._run_request(request)
    532         check_for_errors(response)
--> 533         return response['result']
    534 
    535     def _request_notify(self, methodname, params, rpcid=None):

TypeError: 'NoneType' object is not subscriptable

In [57]: 
```

### Description

When doing relay_set over jsonrpc, the call raises TypeError. Apparently because there is no result object in the response from the server.

### Steps to Reproduce

1. Install evok from required revision
2. Do the jsonrpc call

**Expected behavior:** Call finishes without an exception

**Actual behavior:** TypeError raised.

**Comment by tknot** [comment]
This occurs if you do a manual update without updating the tornadorpc_evok library in /etc/lib and/or /opt/evok/evok.  Is it possible you have done so?

This is the specific commit which fixed the problem [a19b057ffc3281b1ca807b66bb4c60e0cd62956a](https://github.com/UniPiTechnology/evok/commit/a19b057ffc3281b1ca807b66bb4c60e0cd62956a
)

I have verified that the behaviour does not occur on a fresh install.

Provisionally marked as not-a-bug awaiting further information

**Comment by jaryn** [comment]
Installing the fresh evok with the `install-evok.sh` helped with this problem thanks.
